t/01-test_needle.t

```diff
@@ -413,6 +413,22 @@ subtest 'click point' => sub {
     is_deeply($needle, undef, 'multiple click points not accepted');
 };
 
+subtest 'workround property' => sub {
+    my $workaround_string_needle = needle->new($misc_needle_dir . 'check-workround-bsc1234567-20190522.json');
+    my $workaround_hash_needle   = needle->new($misc_needle_dir . 'check-workround-hash-bsc7654321-20190522.json');
+    my $no_workaround_needle     = needle->new($misc_needle_dir . 'click-point-center.json');
+
+    is($workaround_string_needle->has_property("workaround"), 1, "workaround property found when it is recorded in string");
+    is($workaround_hash_needle->has_property("workaround"),   1, "workaround property found when it is recorded in hash");
+    is($no_workaround_needle->has_property("workaround"),     0, "workaround property not found");
+
+    is($workaround_string_needle->get_property_value("workaround"), "bsc#1234567", "get correct value when workaround is recorded in string");
+    is($workaround_hash_needle->get_property_value("workaround"), "bsc#7654321: this is a test about workaround.", "get ccorrect value when workaround is recorded in hash");
+    is($workaround_hash_needle->get_property_value("test"),     "", "no test value");
+    is($no_workaround_needle->get_property_value("workaround"), "", "no workaround property");
+    is($no_workaround_needle->get_property_value("glossy"),     "", "glossy property is a string, has no value");
```

Comment:
You could also add a test for `has_property` and `glossy`.

Reply:
Added